ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -852,15 +856,6 @@ tests:
         OPENSHIFT_VERSION=4.12
         OPENSHIFT_INSTALL_RELEASE_IMAGE=""
     workflow: assisted-baremetal
-- as: e2e-metal-assisted-4-13
```

Comment:
maybe we should keep this job only for the ``bump_ocp_upstream.py`` script that triggers the affected jobs (it doesn't have any handling for a default version, it just do ``/test edge-e2e-metal-assisted-4-Y``)

Reply:
Done